content/tbb/export-and-import-bookmarks/contents.lr

```diff
@@ -0,0 +1,57 @@
+_model: question
+---
+title: How can I Export and Import Bookmarks?
```

Comment:
You don't need to capitalize bookmarks.

Please add Tor Browser on title, so random people in the internet know that you're talking about Tor Browser and not other browser.